chapters/data-collection.tex

```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
-Any time anyone accesses the data --
-either when viewing it in the browser or downloading it to your computer --
-they will be asked to provide the key.
-Only project team members named in the IRB are allowed access private keys 
-that can be used to decrypt sensitive data.
 
-\subsection{Storing data securely}
-% doesn't cover 'big' data
-For analysis, you will need to transfer the data from the data collection software to
-a secure location on your computer or a cloud drive.
-While public/private key encryption is optimal for one-way transfer
-from the data collection device to the data collection server,
-it is not practical once you start interacting with the data.
-Instead, at this stage you need \textbf{symmetric encryption}\sidenote{
-  \url{https://dimewiki.worldbank.org/Encryption\#Symmetric\_Encryption}}.
-In this set-up, a single key is used to both encrypt and decrypt the information.
-Since only one key is used, the workflow can be simplified:
-re-encryption after decrypted access can be done automatically,
-and the same secure folder can be used for multiple files.
-Data should be stored in an encrypted folder,
-using, for example, VeraCrypt.\sidenote{
-	\url{https://www.veracrypt.fr} and \url{https://github.com/worldbank/dime-standards/blob/master/dime-research-standards/pillar-4-data-security/data-security-resources/veracrypt-guidelines.md}}
-These files can be interacted with and modified like any unencrypted file,
- as long as you have the key.
-The following workflow allows you to receive data and store it securely,
-without compromising data security:
+Data encrypted this way can securely pass through the server owned by the data collection software provider,
+and you should never decrypt it until it is downloaded on your computer.
+If your data collection service allows you to browse data in the browser, 
+then the encryption is only implemented correctly if you are asked for the key each time.
+
+Finally, the data security standards that apply when receiving confidential data from the field
+also apply when transferring confidential data to the field.
+In some survey software, 
+you can use the same encryption that allows you to receive data securely
+from the field, to also send confidential data,
+such as an identifying list of respondents, to the field.
+
+
+\subsection{Receiving or sharing data securely}
+
+If you receive confidential data from a partner organization 
+or share confidential data with your research team over the internet,
+it must always be encrypted.  
+The best practice is to reduce the amount of confidential data
+needed to be shared or transfer.
+If you do not need the confidential information you will simplify your workflow
+by collecting or receiving data without any confidential information in the first place.
+This should always be your first option when possible.
+
+Usually we need to deal with confidential data at some point in the project, 
```

Comment:
For the final review, we need to make sure we use the terms 'confidential' and 'personally-identifying" consistently throughout all sections/chapters

Reply:
Ok, opened up an issue for that here: https://github.com/worldbank/d4di/issues/454